### PR TITLE
Set up storybook tool bar for dark + light mode swap.  

### PR DESCRIPTION
**Pull request by Balance8**
## Description

Set up storybook tool bar for dark + light mode swap.  Allow chromatic to take screen shots of both

## Checklist

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, if necessary.
- [x] I have added tests that prove my fix is effective or my feature works.
- [x] New and existing tests pass locally with my changes.

## Screenshots (if applicable)

image
image
image
image
image

### CODE REVIEW

File: apps/storybook/.storybook/preview.tsx [hunk 1]
```diff
@@ -13,30 +13,51 @@ const preview: Preview = {
...
-    (Story) => (
-      <ThemeProvider>
-        <TooltipProvider>
-          <Story />
-        </TooltipProvider>
-        <Toaster />
-      </ThemeProvider>
-    ),
+    (Story, context) => {
+      if (typeof document !== 'undefined') {
```
Comment (Balance8) [inline apps/storybook/.storybook/preview.tsx:46]: If anyone can think of a better way to do this I am happy to hear it

**Comment by Balance8** [comment]
I just learned I might be able to do this using `import { withThemeByClassName } from '@storybook/addon-themes';`

**Comment by haydenbleasel** [comment]
@Balance8 Nice work! Yeah if you want to check out `/addon-themes`, it could be the way to go 🤷

**Comment by Balance8** [comment]
> @Balance8 Nice work! Yeah if you want to check out `/addon-themes`, it could be the way to go 🤷

all good to go now:
image
image

**Comment by Balance8** [comment]
The autogen docs are a bit tricky to configure, the best I could get was this for dark mode since the padding is a controlled storybook component

image
image

**Comment by Balance8** [comment]
@haydenbleasel 

Would you like me to add:

```json
    "build-storybook": "storybook build",
    "chromatic": "chromatic --exit-zero-on-changes"
```

to the `package.json`.  `build-storybook` is the autorecognized command from chromatic and `chromatic` is the command they use to auto publish.

**Comment by haydenbleasel** [comment]
@Balance8 Yes good call, adding those commands would be a good idea 👍

**Comment by Balance8** [comment]
> @Balance8 Yes good call, adding those commands would be a good idea 👍

@haydenbleasel done ☑️

**Pull request merged**